packages/framework/src/Model/Order/Item/OrderItemPriceCalculation.php

```diff
@@ -57,6 +57,19 @@ public function calculatePriceWithoutVat(OrderItemData $orderItemData): Money
         return $orderItemData->priceWithVat->subtract($vatAmount);
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Order\Item\OrderItem $orderItem
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
+     */
+    public function getTotalPrice(OrderItem $orderItem): Price
```

Comment:
Here my reasoning was: either you want to explicitly *calculate* the price, so you use the `calculateTotalPrice` or you just want to *get* the total price consistently with the application logic (which means it can be forced, or is recalculated on-the-fly as it was previously). Method `calculateTotalPriceIfNotForced` doesn't really sound to me as a method that should be used in majority of use-cases, it sounds more like an edge case...

Reply:
ok, I wanted to give a suggestion.
`getTotalPrice` doesn't express it might calculate the price and also might return already calculated price. When should the implementator use `calculateTotalPrice` and when `getTotalPrice`? This naming doesn't help

Reply:
I think we should keep only calculateTotalPrice and it should do exactly what getTotalPrice does now. As @simara-svatopluk mentioned - the naming right now does not help in any way and therefore I would prefer to keep it as it was.

Reply:
Oh... maybe it makes more sense now that https://github.com/shopsys/shopsys/pull/1129/commits/3cf8acd966cc0e8ec794f7c3aab9b2aa8dbfc9a6 was merged.

It would definitely make the change much much simpler.

Reply:
changed in 5865c939e

Reply:
I would like to squash some commits together as there are a lot of changes that revert of modify something built in previous commits... But I'd like to do that only after the changes are code-reviewed.